### DISCUSSION TITLE
REST API Connector - Pass additional headers

### DISCUSSION THREAD

**Discussion by SantoshSrinivas79** [Budibase/budibase#3934] [Help] [answered]
In my self-hosted reverse proxy, I am passing along a few custom headers using [this on NGINX](https://github.com/openresty/headers-more-nginx-module#synopsis).

Is there a way to pass along these headers with the RESI API connector calls?

**Comment by mjashanks** [comment, marked as answer]
Not at the moment. This is quite similar to https://github.com/Budibase/budibase/issues/3827

It would be great to have Request headers and other properties available for binding.

**Reply by SantoshSrinivas79** [reply]
@mjashanks is this something that could be addressed in the short term? this could really solve a lot of issues

**Reply by jlangree** [reply]
I agree 100%. If there were any sort of workaround it would be extremely helpful.

**Reply by jmxt3** [reply]
This ill be a game-changer.
